compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/CoreTextField.kt

```diff
@@ -304,6 +304,8 @@ internal fun CoreTextField(
     val coroutineScope = rememberCoroutineScope()
     val bringIntoViewRequester = remember { BringIntoViewRequester() }
 
+    rememberClipboardEventsHandler(manager, state.hasFocus)
```

Comment:
what worries me that we are introducing a platform-specific hook to the common so that we do something on one platform. I do realise that this is sometimes unavoidable, all I'm trying to say that before it's unavoidable it worth to invest into investigating any non-invasiva alternative/

Reply:
I looked once again and I reach a conclusion that to implement this feature, we can't avoid changing common. Here is what I base my conclusion on:
- the events handler needs to "know" some textfield state (what text field is active, what text is selected, etc.). In this case the necesary state is provided by TextFieldSelectionManager (it's internal in foundation module)
- there is no global state of text fields. TextFieldState and TextFieldSelectionManager are created (and remembered) for every CoreTextField during its initialization. It makse CoreTextField a convenient place to initialize our event handler too.
- An alternative approach to find a focused node and to dispatch an event to it would require changes in common too, because currently the nodes can handle just key events, but not higher-level clipboard events.


There're similar cases when one platform has a feature/function implemented, while all other platfroms have just empty function body. 
The fact that we change the common code here, doesn't require us to urgently upstream this change, because it's an internal implementation detail which doesn't affect android behaviour.